.dockerignore

```diff
@@ -10,7 +10,6 @@ node_modules/svg-sprite-loader/examples
 .git*
 .storybook
 stories
-tests
```

Comment:
I don't think you want to commit anything under `node_modules`. Are you sure this is needed?